lib/master.js

```diff
@@ -134,6 +150,10 @@ class Master extends BaseService {
             time: null,
             killed: true
         };
+
+        worker.removeListener('message', this.workerMessageHandlerMap[worker.process.pid]);
+        worker.removeListener('exit', this.workerExitHandlerMap[worker.process.pid]);
+
```

Comment:
Haven't we just removed the same handler on the line 140?

Reply:
When you are removing these handlers, you are not cleanin them out of the map. memory leak.

Reply:
Fixed